.travis.yml

```diff
@@ -45,17 +46,30 @@ cache:
 
 # Not needed since we use Puppeteer in karma.conf.ci.js
 # It downloads Chrome itself and works with or without Travis
-#addons:
 #  chrome: stable
 
 script:
   - npm run lint:all
   - npm run test:ci:all
+  # Serve site as background process
+  - cd showcase && npm run server:prod:ci &
+  # Run e2e tests
+  - npm run test:showcase:browserstack
+  # Kill http-server
+  - pkill -f http-server
   - npm run docs:coverage
   - if [[ ${TRAVIS_TAG} == "" && ${TRAVIS_EVENT_TYPE} != "cron" ]]; then npm run build:showcase:ghpages:ci-test-env; else npm run build:showcase:ghpages; fi
   - npm run docs:publish
   - npm run release:publish
   - bash ./scripts/ci/print-logs.sh
 
+addons:
+  browserstack:
+    forcelocal: true
+    username: carlonomes1
+    access_key:
+      secure: "NZ7TXMnHXokMitIXKb42TVcc3fmpLCK+A3FlN2BBhVsZBFeUxj1g5isjOmTAqq+6/pdI6cao0dB7ylq930rrXFz81/ghvzeX/JJrWPLO7ts6Sj9lavJFj1TL/ksOu16GFf0MvSGYfXpXCnIhG5YONs7/4MyohT+xg9mFrUhHseUV+dFHE8IBSSyEmsoIcyJOI6kMWyaHVI0hIBJc/VKqQkMw9LG6ENRy9BphHAbE1onxx2K20SLoyg7/n2qjlptWW5+IU9qkWTLKdpa6S68PXz6woKKrjTObyW93Cv0YLhZxpPFPkS2z8Mmm4NstnVlzuVs2QqPMsFykMl8weK9vRQjLtBClizicTUGvZr4jkfKPfOFgzyX9LsWTdPuzmfgq0pMU51YFu7BSHx92nZ5U8Gi8A/4EuKLWCoQzS4CIRgpriqkXbE76jZABWm2DuL86nWOzVNi80NM+wqlpDs/6W5BtBGPdmDiZJ9hsLFLKuRaS6yXnWNi9v4GL4XlUb5g4KNQYbrcrlX4H5LVRk9/bH9trM4kV64D0fd0AWq5UPeckK+3tiKoy4VP+gVk0n1R/u7iKS4iGtKZKL9FJpcmoPLCNu0QCIUgdF89VPGnCiZ0zHJp1xd5wVJ+VNA9nlLJXaNSEsg03P5P0psyeSQnM7AAYyCZYl4TbeHq7aoorv1g="
```

Comment:
It is the encrypted BrowserStack access Key. It would be cleaner to store it as an environmental. But I could not get that to work. The same for the username.

Reply:
I tried all the ways I could think of to put these in an environmental but to no avail. 😞

Reply:
🤷‍♂️ https://github.com/travis-ci/travis-ci/issues/1946